srcpkgs/python3-adblock/template

```diff
@@ -0,0 +1,51 @@
+# Template file for 'python3-adblock'
+pkgname=python3-adblock
+version=0.4.2
+revision=1
+wrksrc="${pkgname/python3/python}-${version}"
+build_style=python3-pep517
+build_helper="rust"
+hostmakedepends="maturin pkg-config cargo libressl-devel"
+makedepends="libressl-devel python3-devel"
```

Comment:
I have confirmed, `libressl-devel` is necessary for target as well. You probably already knew that.